### ISSUE TITLE
Removing anaconda environments on ZFS datsets results in errors

### ISSUE THREAD

**Issue by JMoVS** [openzfsonosx/zfs#378]
I installed [Anaconda](http://continuum.io/downloads#py34) into a zfs dataset. This worked.

Then I updated conda with `conda update conda`, which also worked fine.

Then I created a new environment:
`conda create --name foo python=3`
which still worked.
activate it shortly with ```source activate foo``` and deactivate it again with `source deactivcate`

now try removing that environment:
`conda remove --name foo --all`
it removes it. Now try creating a new conda environment:
`conda create --name bar python=3'

it will reliably result in an error claiming that ssl is not available.

When I do the same steps in anaconda installed on HFS+, it doesn't produce that error

**Comment by JMoVS** [comment]
Some log messages that might be of interest (appeared a couple of times):
`14.09.15 13:36:14,148 mds[63]: (FMW.Warning:1078) event:1 had an arg mismatch.  ac:2 am:51`
not many but more than 10

and a heck of a lot of the following messages:
`14.09.15 13:34:11,499 mds[63]: (FMW.Error:387) WE ARE DROPPING FMW EVENTS!
`

what I did was installing anaconda and deleting and etc (which somehow involves 150.000 files in the directory). But I did the same to my internal HFS volume and that did survive it.
